include/zephyr/linker/common-ram-end.ld

```diff
@@ -0,0 +1,16 @@
+/*
+ * Added after the very last allocation that might land in RAM to define the various
+ * end-of-used-memory symbols
+ */
+
+    SECTION_PROLOGUE(.last_ram_section,,)
+    {
+#ifdef LAST_RAM_ALIGN
+	LAST_RAM_ALIGN
+#endif
+	_image_ram_end = .;
+	_end = .; /* end of image */
+	z_mapped_end = .;
+    } GROUP_NOLOAD_LINK_IN(RAMABLE_REGION, RAMABLE_REGION)
+
+    GROUP_END(RAMABLE_REGION)
```

Comment:
I wonder if it would be better to keep `GROUP_END(RAMABLE_REGION)` in the main linker scripts so that there exist the matching `GROUP_START` and `GROUP_END` for the `RAMABLE_REGION` in the same file.

Reply:
yeah, probably a better plan as it leaves the file properly "bracketed".